.circleci/config.yml

```diff
@@ -2,37 +2,36 @@ version: 2
 jobs:
   build:
     environment:
-      GRADLE_OPTS: -Dorg.gradle.jvmargs="-Xmx2048m -XX:+HeapDumpOnOutOfMemoryError" -Dkotlin.compiler.execution.strategy=in-process
```

Comment:
Only two changes on this file:
1) this now defaults to what Docker reports as available in the CI. Also, some were specified through the `build.gradle` files instead